historical/vpc/poller.py

```diff
@@ -36,18 +36,18 @@ def handler(event, context):
     log.debug('Running poller. Configuration: {}'.format(event))
 
     for account in get_historical_accounts():
```

Comment:
It seems like we are doing a lot of:
```
for account in get_historical_accounts():
      for region in POLL_REGIONS:
```

I wonder if this could be refactored into:
```
for account, region in get_historical_account_regions():
```